src/tests/multihost/alltests/test_multidomain.py

```diff
@@ -321,19 +321,18 @@ def test_0012_filesldap(self, multihost, multidomain_sssd):
             getent = 'getent passwd %s' % user
             cmd = multihost.client[0].run_command(getent, raiseonerr=False)
             assert cmd.returncode == 0
-        for provider in ['files', 'ldap1']:
-            ldb = 'ldbsearch -H /var/lib/sss/db/config.ldb -b '\
-                  'cn=%s,cn=domain,cn=config' % (provider)
-            cmd = multihost.client[0].run_command(ldb, raiseonerr=False)
-            if cmd.returncode == 0:
-                if provider == 'ldap1':
-                    provider = 'ldap'
-                checks = ['enumerate: True', 'id_provider: %s' % provider,
-                          'max_id: [0-5]010', 'min_id: [0-5]000']
-                for str1 in checks:
-                    find = re.compile(r'%s' % str1)
-                    result = find.search(cmd.stdout_text)
-                    assert result is not None
+        provider = 'ldap1'
+        ldb = f'ldbsearch -H /var/lib/sss/db/config.ldb -b cn={provider},cn=domain,cn=config'
+        cmd = multihost.client[0].run_command(ldb, raiseonerr=False)
+        if cmd.returncode == 0:
+            if provider == 'ldap1':
+                provider = 'ldap'
+            checks = ['enumerate: True', f'id_provider: {provider}',
+                      'max_id: [0-5]010', 'min_id: [0-5]000']
+            for str1 in checks:
+                find = re.compile(str1)
+                result = find.search(cmd.stdout_text)
+                assert result is not None
 
     @pytest.mark.tier2
     def test_0013_filesldap(self, multihost, multidomain_sssd, localusers):
```

Comment:
can you please update the name as `test_0013_filesldap` to `test_0013_proxyldap`